crates/gitbutler-core/src/ops/reflog.rs

```diff
@@ -20,7 +20,7 @@ use crate::projects::Project;
 /// <target branch head sha>                 <oplog head sha>
 ///
 /// The reflog entry is continuously updated to refer to the current target and oplog head commits.
-pub fn set_reference_to_oplog(
+pub(crate) fn set_reference_to_oplog(
```

Comment:
this is cool i wasnt familiar with this syntax. The `reflog` is not exported in ops/mod.rs - Is there a way to make it visible only within the ops module? I guess it's not important, im just curious what's possible

Reply:
Yes, there even is a syntax for that named `pub(in <module>)`. This I use quite rarely and instead go with `pub` while possibly putting it into a `mod` (private)` to prevent it to leak. Now that I say that, the `pub(crate)` portions could probably use another round, this time with `cargo doc` to help visualize what's visible to see if it *should* be visible.